osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -133,6 +135,7 @@ public void ShowComments(CommentableType type, long id)
             if (!IsLoaded)
                 return;
 
+            commentCounter.Current.Value = 0;
```

Comment:
Should probably reside in `clearComments`. (where the comments amount would really become 0)

Reply:
Looks bad visually. `clearComments` called on every sort change

Reply:
Then there should probably be an in-line comment saying that.